RELEASE.md

```diff
@@ -42,6 +50,7 @@
 
 ## Upcoming deprecations for Kedro 0.19.0
 * The Kedro IPython extension will no longer be available as `%load_ext kedro.extras.extensions.ipython`; use `%load_ext kedro.ipython` instead.
+* `kedro jupyter convert`, `kedro build-docs`, `kedro build-reqs` and `kedro activate-nbstripout` will be deprecated.
```

Comment:
Please have a look at this section, as I found that some PRs missed mentioning the changes in `release.md`.

Reply:
This one is minor but missing:
- https://github.com/kedro-org/kedro/pull/1851/files

Reply:
Above "* Use default `False` value for rich logging `set_locals`, to make sure credentials and other sensitive data isn't shown in logs."

should actually be: "* Use default `False` value for rich logging `show_locals`, to make sure credentials and other sensitive data isn't shown in logs."